services/api.tsx

```
@@ -0,0 +1,95 @@
+const getBaseUrl = () => {
+  let baseURL = 'http://localhost:5286';
+
+  console.log('Platform:', Platform.OS);
+
+  switch (Platform.OS) {
+    case 'web':
+      baseURL = 'http://localhost:5286';
+      break;
+      
+    case 'android':
+      const androidIP = getExpoDebuggerIP();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)